Libraries/PeriphDrivers/Source/CAMERAIF/cameraif_reva.c

```diff
@@ -51,16 +51,6 @@
 
 int MXC_PCIF_RevA_Init(void)
 {
-    MXC_SYS_ClockEnable(MXC_SYS_PERIPH_CLOCK_PCIF);
-    //MXC_GPIO_Config (&gpio_cfg_pcif_P0);
-    //MXC_GPIO_Config (&gpio_cfg_pcif_P1);
-    //MXC_GPIO_Config (&gpio_cfg_pcif_hsync);
-    //MXC_GPIO_Config (&gpio_cfg_pcif_vsync);
-    //MXC_GPIO_Config (&gpio_cfg_pcif_pclk);
-    //MXC_GPIO_Config (&gpio_cfg_pcif_pwrdwn);
-
-    //MXC_GPIO_OutClr (gpio_cfg_pcif_pwrdwn.port, gpio_cfg_pcif_pwrdwn.mask);
-
     return 0;
```

Comment:
It looks like none of the parts with the PCIF (AI85 and AI87) uses this `MXC_PCIF_RevA_Init` function anymore. It may be a good idea to just return `E_NOT_SUPPORTED` until this function needs to be implemented in the future.

Reply:
Let's keep it as it is. 
Even if the function is empty returning success  will be better than returning non-success value.